docs/QuickStart-GettingStarted.md

```diff
@@ -1,10 +1,10 @@
 ---
-id: getting-started
+id: quick-start-getting-started
 title: Getting Started
 layout: docs
 category: Quick Start
-permalink: docs/getting-started.html
-next: tutorial
+permalink: docs/quick-start-getting-started.html
```

Comment:
The organization of the docs is not great in my opinion. I am not a big fan of everything being under one mammoth URL level. Optimally, I want to categorize them in proper sub-levels. (e.g., `/docs/quick-start/getting-started.html).

So at some point I am going to push for URL changes.

But I guess for now, I can just stick with the file rename so that we can at least categorize them accordingly for those contributing to the docs.